app/routes/docs.sample.xml.md

```diff
@@ -0,0 +1,54 @@
+---
+handle:
+  breadcrumb: XML
+---
+
+# XML and MIME Data
+
+Here is a collection of functions and example code that may be useful when working with XML and Multipurpose Internet Mail Extensions (MIME) formats.
+This guide provides essential functions to parse XML alerts into dictionaries and structured data formats using Python.
+
+Within your consumer loop, use the following functions to convert the
+content of `message.value()` into other data types.
+
+The `xmltodict` package is not part of the Python standard library. You must install it by running:
+pip:
+
+```sh
+pip install xmltodict
+```
+
+```python
+import xml.etree.ElementTree as ET
+import xmltodict
+
+# Convert a raw XML string into an ElementTree XML object
+def parse_voevent_alert_to_xml_root(message_value):
+    return ET.fromstring(message_value)
+
+# Convert XML string into a Python dictionary
+def parse_voevent_alert_to_dict(message_value):
+    return xmltodict.parse(message_value)
+```
+
+## Saving Alert Data
+
+Use the following to save the raw alert message data to a local file:
+
+```python
+def save_alert(message):
+    with open('path/to/your/file', 'wb') as file:
+        file.write(message.value())
+
+```
+
+## MIME-Encoded Messages
+
+The following function uses the `email` library to parse MIME-encoded messages.
```

Comment:
This is the one and only place that we have ever described "MIME-encoded messages".

I think that this code is supposed to parse the GCN Classic text format.